backend/alexa_tutorial.md

```diff
@@ -0,0 +1,71 @@
+##Steps to building an alexa skill:
+
+###Prerequisites:
+In order to develop an Alexa skill you'll need access to 2 accounts.
+
+1. **aws.amazon.com**: An IAM user with access to the webservices@intrepid.io account
+2. **developer.amazon.com**:  webservices@intrepid.io for amazon developers, this login info is in LastPass.
+
+### Intro
+There are two parts to building an echo skill. (1) Developing the logic for skill. Alexa can be configured to hit any api endpoint. A common practice to set up a lambda function for the Alexa skill to hit. Which is what we'll do in this tutorial.
+(2) Configuring your Alexa skill, this is where you'll create the commands that Alexa will respond to, define the variables you'll have access to in your endpoint and publish your skill.
+
+### Setting up your lambda
+
+1. Login using your IAM user and ensure that you're in the correct region
```

Comment:
is a link appropriate here? I know when i tried to do this i was swimming in aws dashboards. one reason not to do it might be because those links might change? just a thought.